deployment/terraform/participant/main.tf

```
@@ -273,6 +273,11 @@ resource "azurerm_storage_blob" "did" {
         "type" : "IdentityHub",
         // Only the query parameters are used, see MockCredentialsVerifier class
         "serviceEndpoint" : "http://dummy?region=${urlencode(local.connector_region)}"
+      },
+      {
+        "id" : "#ids-url",
+        "type" : "IdsUrl",
```

```suggestion
        "type" : "IDSMessaging",
```

Since this is the definition of a Service, we shouldn't have "URL" in the service type.

ref https://github.com/International-Data-Spaces-Association/IDS-Messaging-Services

---

if we do this then we need to do the same in local setup did.json. Just adding as a comment to not miss this.

---

done, I used IDSMessaging in all places